tests/test_sindri.py

```diff
@@ -0,0 +1,111 @@
+import os
+import time
+
+from src.sindri.sindri import Sindri
+
+
+# Load sample data.
+# This assumes the sindri-resources repo is cloned in the parent dir of this repo
```

Comment:
Might want to use a git submodule if the tests depend on it?

Reply:
I think we can hold off with the git submodule if we plan on recording API responses in #23